kDriveCore/Data/Models/Drive/Drive.swift

```diff
@@ -164,24 +118,36 @@ public class Drive: Object, Codable {
         return _categoryRights ?? CategoryRights()
     }
 
+    public var capabilities: DriveCapabilities {
+        return _capabilities ?? DriveCapabilities()
+    }
+
+    public var pack: DrivePack {
+        return _pack ?? DrivePack()
+    }
+
     public var isUserAdmin: Bool {
         return role == "admin"
     }
 
+    public var isDriveUser: Bool {
+        return role != "none" && role != "external"
```

Comment:
Same comment about "none" here.

Reply:
Ideally `role` should be placed in an enum but as I haven't consulted the back-end on the possible values it is like this for now.

Reply:
ok perfect, I approve the PR then. Thanks.